CMakeLists.txt

```diff
@@ -153,6 +153,17 @@ set(Slicer_EXTENSION_SOURCE_DIRS
 #  )
 #list(APPEND Slicer_EXTENSION_SOURCE_DIRS ${${extension_name}_SOURCE_DIR})
 
+#SlicerMorph modules
+set(extension_name "SlicerMorph")
+set(${extension_name}_SOURCE_DIR "${CMAKE_BINARY_DIR}/${extension_name}")
+FetchContent_Populate(${extension_name}
+ SOURCE_DIR     ${${extension_name}_SOURCE_DIR}
+ GIT_REPOSITORY ${EP_GIT_PROTOCOL}://github.com/andinet/SlicerMorph.git
```

Comment:
Ideally, do not associate custom application with personal fork. 

Instead, consider the following:

# Case 1:

There are no modification specific to your project, simply use the official GitHub organization and project name

# Case 2

You have modification specific to your project

Generally speaking, consider the following convention  for branches: 

```
<projectname>[-x.y.z]-YYYY-MM-DD-SHA{9}
```

where 
* `<projectname>` would be `vpaw` 
* `x.y.z`: No version in this case because SlicerMorph is not tagged.
* `YYYY-MM-DD`: date of the last commit common to both your fork and upstream
* `SHA{9}`: First N characters of the SHA of  the last commit common to both your fork and upstream


For examples, see:
* https://github.com/Slicer/VTK/
* https://github.com/KitwareMedical/Slicer
* https://www.slicer.org/wiki/Documentation/Nightly/Developers/ProjectForks  (would need to be updated and migrated to readthedocs)